ui/src/common/components/LiteratureQueryLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Link } from 'react-router-dom';
+import { LITERATURE } from '../routes';
+
+const LiteratureQueryLink = ({ children, recordId }) => (
```

Comment:
I would make a component that has also the IconText part since it is also shared and just get the text as prop

Reply:
Actually, I just saw your comment that it is the same as ```CitationLinkAction```. I think you can generalize that one, we actually could use the formattedNumber in the other places as well

Reply:
Yes, I think that's the best idea